Gemfile.lock

```diff
@@ -662,7 +677,7 @@ GEM
     xml-simple (1.1.4)
     xpath (2.0.0)
       nokogiri (~> 1.3)
-    zeus (0.15.4)
+    zeus (0.15.13)
```

Comment:
Is this change in `Gemfile.lock` also reflected in `Gemfile`? If we need a specific version of `zeus` please specify that in `Gemfile`